thirdparty/cmdconfig/commands/cmdeval/cmdeval.go

```diff
@@ -21,7 +21,7 @@ import (
 func GetEvalFnRunner(name string) *EvalFnRunner {
 	r := &EvalFnRunner{}
 	c := &cobra.Command{
-		Use:     "eval [DIR]",
+		Use:     "eval [DIR | -<STDIN>]",
```

Comment:
Do we have to say `STDIN` here ?

Reply:
What are you suggesting ?

`eval [DIR]` or
`eval [DIR | -]`

Reply:
I am thinking.. just saying `eval [DIR | -]` should be enough..right ?

Reply:
Done.